manager/testdata/new-blueprints/network-policy-module.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: network-policy
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+spec:
+  description: "This module knows how to configure a kubernetes network policy"
+  multi-user: true
+  confChart:
+    name: ghcr.io/mesh-for-data/networkpolicy-isolation:0.1.0
+    values: {}
+  statusIndicators:
+  kind: configuration
+  capabilities:
+  - capability: isolation
+    scope: dataset
```

Comment:
I would say it may have many scopes. Let me list some examples:
1. A Blueprint has steps that deploy an arrow flight service per data asset (like current implementation) -> It's dataset scoped
2. A Blueprint has steps that configure a common flight service -> Might seem workload related but when you think about e.g. the example where a common read service gets connected with a common transform service then the isolation definition between those services would be the same for every workload. This it would be cluster scoped.